_oasis

```diff
@@ -52,3 +52,10 @@ Executable "biokepi-demo"
   CompiledObject: best
   BuildDepends: biokepi, threads
   MainIs:     main.ml
+
+Document biokepi
+  Title:                API reference for Biokepi
+  Type:                 ocamlbuild (0.4)
+  BuildTools:           ocamlbuild, ocamldoc
+  XOCamlbuildPath:      src/lib
+  XOCamlbuildLibraries: biokepi
```

Comment:
I switched to using Oasis because it seems consistent to keep all the capabilities in one place, and the default one that I found in the Makefile didn't work right away (so I assumed it was legacy).